lib/uvm/vm-execute.js

```diff
@@ -61,7 +62,8 @@
         lib = requires[i];
         // Play around with the require cache
         try {
-            libraries[lib.object] = __require(lib.require);
+            required = __require(lib.require);
+            libraries[lib.object] = lib.require === 'lodash3' ? required.noConflict() : required;
```

Comment:
instead of adding a string like this and creating another special case, why not check if noConflict exists and is a function?